### PR TITLE
Documentation for vSphere cleanup

### PR DESCRIPTION
**Pull request by vavuthu**
Signed-off-by: vavuthu <vavuthu@redhat.com>

### CODE REVIEW

**Review by Avilir — APPROVED** [review]

**Review by wusui — DISMISSED** [review]
Looks good to me.

File: docs/usage.md [hunk 1]
```diff
@@ -10,12 +10,24 @@ Run `report-version --cluster-path /my-cluster/path --log-level INFO` to
 get information on various image version deployed by ocs-ci
 
 # usage for cleanup
+
+#### AWS
```
Comment (petr-balogh) [inline docs/usage.md:14]:
```suggestion
## AWS
```

File: docs/usage.md [hunk 2]
```diff
@@ -10,12 +10,24 @@ Run `report-version --cluster-path /my-cluster/path --log-level INFO` to
...
   This should be used only when your cluster-dir is accidentally deleted
   `ci-cleanup [-h] --cluster CLUSTER`
   CLUSTER points to cluster tag eg: mycluster-ocs-ci-jlgzn , without additional
    -master or -worker. This can be found by logging into AWS, Selecting the VM
    and clicking on Tags.
 
+#### vSphere
```
Comment (petr-balogh) [inline docs/usage.md:21]:
```suggestion
## vSphere
```

**Review by petr-balogh — COMMENTED** [review]
Otherwise it looks OK to me. Thanks!

**Comment by petr-balogh** [comment]
Adding WIP to prevent to merge this.  Please remove once fixed my comment.

File: docs/usage.md (hunk 1 above)
Comment (vavuthu) [reply docs/usage.md]: done

File: docs/usage.md (hunk 2 above)
Comment (vavuthu) [reply docs/usage.md]: Done

**Review by petr-balogh — APPROVED** [review]

**Review by PrasadDesala — APPROVED** [review]
LGTM

**Pull request merged**